stories/virtru/securitySettings/securitySettings.jsx

```diff
@@ -0,0 +1,42 @@
+import 'antd/dist/antd.less';
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { SecuritySettingsToggles } from '@';
+
+storiesOf('virtru/securitySettings', module).add(
+  'securitySettings',
+  () => {
+    const showFilesSettings = true;
+    const securitySettings = {
+      expirationDateTime: '13 Aug, 2022',
+      watermark: true,
+      persistentProtection: false,
+    };
+
+    const securitySettingsUpdating = false;
+    const securitySettingChangeHandler = () => {
+      console.log('securitySettingChangeHandler click');
+    };
+
+    return (
+      <SecuritySettingsToggles
+        showFilesSettings={showFilesSettings}
+        expirationDateTime={securitySettings.expirationDateTime}
+        watermark={securitySettings.watermark}
+        persistentProtection={securitySettings.persistentProtection}
+        disabled={securitySettingsUpdating}
+        onChange={securitySettingChangeHandler}
```

Comment:
It could be good to use [knobs](https://storybook.js.org/addons/storybook-addon-knobs-color-options) here. Here is an example: https://github.com/virtru/virtuoso-design-system/pull/360/files#diff-4c73f8ce153a001fd2f7139d8d8ae3c3c9baf54b9e20a0fae61aae019a35ee24